src/core/component.js

```diff
@@ -136,6 +126,38 @@ Component.prototype = {
     return utils.extend({}, data);
   },
 
+  updateCachedValue: function (value, isSinglePropSchema) {
+    var parsedValue;
+    if (value === '') {
+      this.attrValue = undefined;
+      return;
+    }
+    if (typeof value === 'string') {
+      if (isSingleProp(this.schema)) {
+        parsedValue = this.schema.parse(value);
+        // To avoid bogus double parsings. Like for example
+        // when parsing a src id to it's url #monster -> models/monster.dae
+        if (typeof parsedValue === 'string') { parsedValue = value; }
```

Comment:
If we parse the cached value then ```#monster``` turns in to ```models/monster```. When calculating the final estate of the component via buildData, ```models/monster``` will be parsed again following the squema. That's not what you want.